modules/configuring-assumerole-acs.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * integration/integrate-with-image-registries.adoc
+:_module-type: PROCEDURE
+[id="configuring-assumerole-acs_{context}"]
+= Configuring AssumeRole in {product-title-short}
+
+[role="_abstract"]
+After configuring AssumeRole in ECR, you can integrate {product-title} with Amazon Elastic Container Registry (ECR) by using AssumeRole.
+
+.Procedure
+. On the RHACS portal, navigate to *Platform Configuration* -> *Integrations*.
+. Under the *Image Integrations* section, select *Amazon Elastic Container Registry*.
++
+The *Configure image integration* modal box opens.
+. Click *New Integration*.
+. Enter the details for the following fields:
+.. *Integration Name*: The name of the integration.
+.. *Registry ID*: The ID of the registry.
+.. *Region*: The region for the registry.
+. If you have configured AssumeRole with container IAM leave the *Use container IAM role* option checked. Otherwise, uncheck the *Use custom IAM role* and enter the details for the following fields:
+.. *Access key ID*: The access key ID for the role.
+.. *Secret access key*: The secret access key for the role.
+. Select the *Use AssumeRole* checkbox.
+. Enter the details for the following fields:
+.. *AssumeRole ID*: The ID of the role to assume.
+.. *AssumeRole External ID* (optional): If you are using an link:https://docs.aws.amazon.com/IAM/latest/UserGuide/id_roles_create_for-user_externalid.html[external ID with AssumeRole], you can enter it here.
+. Select *Test* (*`checkmark`* icon) to test that the integration with the selected registry is working.
```

Comment:
@sagidlow I've changed the procedure to reflect changes in the product, can you review this one again?